documentation/MIGRATION_GUIDE.md

```diff
@@ -105,13 +109,16 @@ In the latest version, if a request is a long-running operation, the function na
 future, err := virtualMachinesClient.CreateOrUpdate(context.Background(), "<resource group name>", "<virtual machine name>", param)
 if err != nil {
 	log.Fatal(err)
+    return
```

Comment:
Here is an indent inconsistency.
And do we have to add the `return` here? The `log.Fatal` will exit the program therefore the return here is unreachable

Reply:
Oh, I make a mistake for the function of log.Fatel. I'll remove all the return then.